pa/pa.c

```diff
@@ -253,6 +259,29 @@ void forgetCommand()
     }
 }
 
+/*
+Print list of commands and actions
+*/
+void listCommands()
+{
+    // Get the file path
+    char directory[100];
+    getFilePath(directory, false);
+    FILE *commandFile;
+    commandFile = fopen(directory, "r");
+
+    char command[100];
+    char action[100];
+    while (fscanf(commandFile, fileFormatBoth, command, action) != EOF)
+    {
+        // Print command and action with colors
```

Comment:
Dang! Can you remove this comment since we're not doing colors? After that, this is good to go! 👍